server/src/ClientToDBAPI/ClinicalDetailsRoute/mainRoute.ts

```diff
@@ -19,106 +20,102 @@ import {
 } from '../../DBService/ClinicalDetails/Mutation';
 import { GetInvestigatedPatientClinicalDetailsFields } from '../../Models/ClinicalDetails/GetInvestigatedPatientClinicalDetailsFields';
 
-const complexityCalculationMessage = 'patient clinical details by status';
 const clinicalDetailsRoute = Router();
-const errorStatusCode = 500;
 
 clinicalDetailsRoute.get('/symptoms', (request: Request, response: Response) => {
     const symptomsLogger = logger.setup({
-        workflow: 'Fetching Symptoms',
+        workflow: 'query all symptoms',
         investigation: response.locals.epidemiologynumber,
         user: response.locals.user.id
     });
-    symptomsLogger.info('launching DB request', Severity.LOW);
-    graphqlRequest(GET_SYMPTOMS, response.locals).then((result: any) => {
-        symptomsLogger.info('got respond from DB', Severity.LOW);
-        response.send(result)
+
+    symptomsLogger.info(launchingDBRequestLog(), Severity.LOW);
+    graphqlRequest(GET_SYMPTOMS, response.locals)
```

Comment:
Maybe rename to GET_ALL_SYMPTOMS? not really clear that it's not something like getSymptomsByUserId